src/alpha/AlphaChallengeTab.js

```diff
@@ -92,10 +92,21 @@ export default function AlphaChallengeTab({state, updateState, popup}) {
 
     const challengeBonus = getChallengeBonus(state)
 
+    //allow "Your {challengeBonus.full} challenge completions and {challengeBonus.segment} segment completions" to be
+    //grammatically correct no matter the count of completions
+    let challengeCompletions = "completions"
```

Comment:
Would prefer to have these defined in a more concise way, like:
```suggestion
    const challengeCompletions = (challengeBonus.full === 1) ? "completion" : "completions"
```